### PR TITLE
#165020168 Add ratings

### PR DESCRIPTION
**Pull request by fridolinho**
### What does this PR do?
Add a 5 star rating system.
### Description of Task to be completed?
- Design the UI for ratings
- Add rating functionalities
### How should this be manually tested?

    git clone https://github.com/andela/ah-kgl-avengers-frontend.git
    git checkout ft-article-ratings-165020168
    npm install
    npm start
    Login / Read any article
    Rate article by clicking on stars bellow the article title

### What are the relevant pivotal tracker stories?
[Users should be able to rate articles](https://www.pivotaltracker.com/story/show/165020168)
### Screenshots (if appropriate)
N/A
### Questions:
N/A

### CODE REVIEW

File: src/components/readArticle.jsx [hunk 1]
```diff
@@ -2,6 +2,8 @@ import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import ReactHtmlParser from 'react-html-parser';
+import '../../node_modules/@fortawesome/fontawesome-free/js/all';
```
Comment (ezkemboi) [inline src/components/readArticle.jsx]: Why import this from node_modules?

**Review by ezkemboi — COMMENTED** [review]
Left some comments for this work.

File: src/components/readArticle.jsx (hunk 1 above)
Comment (fridolinho) [reply src/components/readArticle.jsx]: Fixed now, thank you for the feedback @ezrqnkemboi

**Comment by ezkemboi** [comment]
Rebase develop changes and get the tests pass.

**Comment by ezkemboi** [comment]
@niyobobo please review this PR. We need your feedback here.

File: public/index.html [hunk 2]
```diff
@@ -20,6 +20,11 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/material-design-iconic-font/2.2.0/css/material-design-iconic-font.min.css"
     />
 
+    <link
+      rel="stylesheet"
+      href=""
```
Comment (niyobobo) [inline public/index.html]: why did you need to add this tag referencing to nothing?

File: src/redux/action-creators/readArticle.js [hunk 3]
```diff
@@ -9,11 +9,20 @@ import {
 
 const readArticle = slug => (dispatch) => {
   const url = `${process.env.REACT_APP_API}/articles/${slug}`;
+  const ratingUrl = `${process.env.REACT_APP_API}/articles/${slug}/ratings`;
   dispatch({ type: FETCH_ARTICLE_START, payload: {} });
   return axios
     .get(url, optRequest)
-    .then((response) => {
+    .then(async (response) => {
```
Comment (niyobobo) [inline src/redux/action-creators/readArticle.js]: I think here you can stick with then-catch promise or async-await instead of chaining them  for code consistency

**Review by niyobobo — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 2 above)
Comment (fridolinho) [reply public/index.html]: resolved

File: src/redux/action-creators/readArticle.js (hunk 3 above)
Comment (fridolinho) [reply src/redux/action-creators/readArticle.js]: resolved

**Review by ezkemboi — APPROVED** [review]
Good work generally.
